features/kubernetes-nmstate/deploy.sh

```diff
@@ -0,0 +1,9 @@
+KNMSTATE_VERSION="v0.12.0"
```

Comment:
Missing shebang

Reply:
Also, it would be nice to `set -e` to stop when some command fails (just in case)